mobile/src/full/java/org/openhab/habdroid/core/GcmIntentService.java

```diff
@@ -102,12 +103,10 @@ private void sendNotification(String msg, int notificationId) {
         NotificationCompat.Builder mBuilder =
                 new NotificationCompat.Builder(this)
                         .setSmallIcon(R.drawable.ic_openhab_appicon_24dp)
-                        .setContentTitle("openHAB")
-                        .setStyle(new NotificationCompat.BigTextStyle()
-                                .bigText(msg))
+                        .setContentTitle(msg)
+                        .setColor(ResourcesCompat.getColor(getBaseContext().getResources(), R.color.openhab_orange, null))
```

Comment:
`ContextCompat.getColor(this, R.color.openhab_orange)` should do the same thing in a shorter way.